conf/modules.config

```diff
@@ -1,310 +1,398 @@
+            pattern: "*bed"
+        ]
+    }
+
+    withName: 'GATK4_CREATESEQUENCEDICTIONARY' {
+        publishDir = [
+            path: { "${params.outdir}/reference/gatk4" },
+            enabled: "params.save_reference",
+            pattern: "*dict"
+        ]
+    }
+
+    withName: 'MSISENSORPRO_SCAN' {
+        publishDir = [
+            path: { "${params.outdir}/reference/msi" },
+            enabled: "params.save_reference",
+            pattern: "*list"
+        ]
+    }
+
+    withName: 'SAMTOOLS_FAIDX' {
+        publishDir = [
+            path: { "${params.outdir}/reference/fai" },
+            enabled: "params.save_reference",
+            pattern: "*fai"
+        ]
+    }
+
+    withName: 'TABIX_BGZIPTABIX' {
+        publishDir = [
+            path: { "${params.outdir}/reference/target" },
+            enabled: "params.save_reference",
+            pattern: "*bed.gz"
+        ]
+    }
+
+    withName: 'TABIX_DBSNP' {
+        publishDir = [
+            path: { "${params.outdir}/reference/dbsnp" },
+            enabled: "params.save_reference",
+            pattern: "*vcf.gz.tbi"
+        ]
+    }
+
+    withName: 'TABIX_GERMLINE_RESOURCE' {
+        publishDir = [
+            path: { "${params.outdir}/reference/germline_resource" },
+            enabled: "params.save_reference",
+            pattern: "*vcf.gz.tbi"
+        ]
+    }
+
+    withName: 'TABIX_KNOWN_INDELS' {
+        publishDir = [
+            path: { "${params.outdir}/reference/known_indels" },
+            enabled: "params.save_reference",
+            pattern: "*vcf.gz.tbi"
+        ]
+    }
+
+    withName: 'TABIX_PON' {
+        publishDir = [
+            path: { "${params.outdir}/reference/pon" },
+            enabled: "params.save_reference",
+            pattern: "*vcf.gz.tbi"
+        ]
+    }
+}
+
+// MAPPING
+process {
+
+    withName: "BWA.*MEM" {
+        ext.args         = { meta.status == 1 ? '-K 100000000 -M -B 3' : '-K 100000000 -M' }
+        ext.args2        = 'sort'
+        publishDir       = [ enabled: false ]
+    }
+
+    withName: 'INDEX_MAPPING' {
+        publishDir       = [
+            path: { "${params.outdir}/preprocessing/${meta.id}/mapped" },
+            enabled: true,
+            pattern: "*{bam,bai}"
+        ]
+    }
+
+    withName: 'SAMTOOLS_MERGE' {
+        publishDir       = [ enabled: false ]
+    }
+
+    withName: 'SEQKIT_SPLIT2' {
+        ext.args         = "--by-size ${params.split_fastq}"
+        publishDir       = [ enabled: false ]
+    }
+}
+
+// Markduplicates Spark NEEDS name-sorted reads or runtime goes through the roof
+// However if it's skipped, reads need to be coordinate-sorted
+// Spark can be used also for BQSR, therefore check for both
+// Only name sort if Spark for Markduplicates + duplicate marking is not skipped
+if (('markduplicates' in params.use_gatk_spark) && (!params.skip_markduplicates)) {
+    process {
+        withName: "BWA.*_MEM" { ext.args2 = 'sort -n' }
```

Comment:
This then overwrites the above config `ext.args2` for BWAMEM?

Reply:
Yes

Reply:
I'm hoping that we can also remove the `part` thingy from the bwa*mem processes

Reply:
I will think about how to do this. I am sure there is a way

Reply:
Where are you using it? I don't see it in the process anymore

Reply:
https://github.com/nf-core/sarek/blob/dev/subworkflows/nf-core/mapping.nf#L5-L6

Reply:
If it's meant to be part of the `prefix` still this may be a good chance to petition for 
`def prefix = task.ext.prefix?: meta.id`
Then you can use:
```
ext.prefix = { "${meta.id}${params.split_fastq > 1 ? reads.get(0).name.findAll(/part_([0-9]+)?/).last().concat('.') : ""}"
```

Reply:
Oh yes, that's what I wanted to figure it out.
I'll let someone else try that, I'm going to the NGI retreat today, so won't be able to work much

Reply:
yep that is what I thought of, to also pull that in the modules config. I am glad I wasn't completely off here. I'll give it a try probably tomorrow afternoon

Reply:
plus by then maybe the yet new modules syntax is there and I can catch two birds with one stone 😆

Reply:
@FriederikeHanssen will you be adding the split_fastq logic in the bwamem modules in https://github.com/nf-core/sarek/pull/462 or will you be doing that in a separate PR?

Reply:
I'll see that I can add it in :)